Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/locate.h

```diff
@@ -28,6 +28,7 @@
 #include <CGAL/boost/graph/named_params_helper.h>
 #include <CGAL/Random.h>
 #include <CGAL/use.h>
+#include <CGAL/squared_distance_3.h>
```

Comment:
@sloriot can you confirm that these includes should not be needed.

Reply:
I think you need it because the free function is used and the Kernel does not need to be included before including this header.

Reply:
Will it then not be missing in all kind of header files?